client/flutter/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -365,7 +370,9 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
+				CODE_SIGN_ENTITLEMENTS = Runner/Runner.entitlements;
 				CURRENT_PROJECT_VERSION = "$(FLUTTER_BUILD_NUMBER)";
+				DEVELOPMENT_TEAM = 3BY6K99ZR5;
```

Comment:
I assume this is you @cbgrey - your own account ?

Reply:
Yes, I believe so. Automatically entered by XCode.